docs/generic-bots/monkey-bot.mdx

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: 'Monkey Bot'
+---
+
+# Monkey Bot
+
+![Monkey Bot Image](img/monkey-bot.png)
+
+The Regression Games SDK has a built-in **Monkey Bot** that automatically determines the applicable keyboard and mouse inputs for the game under test and randomly performs them as the game is running. The SDK also offers the capability to configure the actions taken by the bot to control the functionality that is exercised by the bot.
+
+## Usage
+
+For most games, the Monkey Bot can be quickly configured and deployed. First, the bot actions are configured through the Action Manager panel. Then, the monkey bot is started through the Regression Games Bot Manager.
+
+### Configuring Bot Actions
+
+Open the Action Manager panel by navigating to **Regression Games > Configure Bot Actions**. You can dock the panel to make it easier to access.
+
+![Configure Bot Actions Menu Item](img/configure-bot-actions.png)
+
+In the panel, press the **Analyze Actions** button. The SDK will now automatically analyze the game code and assets to identify all the device inputs that are applicable to the game.
+
+![Initial Action Manager Panel](img/action-manager-panel-initial.png)
+
+:::info Note
+The automated analysis supports commonly used device input APIs from the Legacy Input Manager, Input System, and Unity UI systems. However, the use of non-standard input handling libraries is not supported. Where possible, the analysis will emit a warning if it is unable to resolve an input.
+:::
+
+After the analysis completes, you will see each device input associated with the type of object that listens for it. You can select an action to view more details about it. The checkbox next to each action can be unchecked to exclude the action from consideration by the bot. For example, you may want to exclude "Quit", "Pause", or "Exit" keys or buttons that may interfere with the bot's ability to progress through the game.
```

Comment:
When it says "you will see each device input associated with the type of object that listens for it", is `object` the right word? This might be confused with game object, but isn't it more about MonoBehaviours?

Reply:
Yeah, that could be confusing. Technically it might not be a MonoBehaviour, e.g. it could be a UnityEngine.UI.Button if it came from Unity UI. To be accurate I've changed this to "component", but do let me know if you think it could be clearer to call it a MonoBehaviour anyways.